services/chat/api.go

```diff
@@ -285,10 +285,6 @@ func (api *API) toAPIChat(protocolChat *protocol.Chat, community *communities.Co
 	}
 	chat.Members = chatMembers
 
-	if chat.CommunityID == "" {
-		chat.CommunityID = pubKey
-	}
```

Comment:
Removed this because even though it makes sense that personal chanels are in the user's "community", having it named `communityId` can cause confusion, as well as causes some breaking changes where some code can look at `communityId` to see if a channel is inside a community